flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/rank/FastTop1Function.java

```diff
@@ -30,38 +30,28 @@
 import org.apache.flink.table.data.RowData;
 import org.apache.flink.table.runtime.generated.GeneratedRecordComparator;
 import org.apache.flink.table.runtime.keyselector.RowDataKeySelector;
+import org.apache.flink.table.runtime.operators.rank.utils.FastTop1Helper;
 import org.apache.flink.table.runtime.typeutils.InternalTypeInfo;
 import org.apache.flink.util.Collector;
 
-import org.apache.flink.shaded.guava32.com.google.common.cache.Cache;
-import org.apache.flink.shaded.guava32.com.google.common.cache.CacheBuilder;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.util.Map;
-import java.util.concurrent.TimeUnit;
-
 /**
  * A more concise implementation for {@link AppendOnlyTopNFunction} and {@link
  * UpdatableTopNFunction} when only Top-1 is desired. This function can handle updating stream
  * because the RankProcessStrategy is inferred as UpdateFastStrategy, i.e., 1) the upsert key of
  * input steam contains partition key; 2) the sort field is updated monotonely under the upsert key.
```

Comment:
NIT: steam -> stream. 
NIT: I am not sure what you are saying by the monotonely sentence - I wonder if you could rephrase please.

Reply:
In fact, I haven't changed those comments, but I think I can update them. 
Regarding `monotonely`, you can find more info in `FlinkRelMdModifiedMonotonicity`. In simple terms, it refers to whether a particular column is monotonically increasing or monotonically decreasing.